app/views/categories/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+  <div class="row">
+    <% @categories.each do |category| %>
+      <div class="row catago" onclick="window.location.href='<%= category.expenses.present? ? expense_path(category.expenses.first) : expenses_path(category_id: category.id) %>';">
+        <div class="col">
+          <div class="image-container">
+          <% if category.icon.attached? %>
+          <%= image_tag category.icon, class: "category-icon img-fluid" %>
+        <% else %>
+          <p>No Image</p>
+        <% end %>
+          </div>
+        </div>
+
+        <div class="col">
+          <p><%= category.name %></p>
+          <% created_date = category.created_at.strftime('%e%p %B %Y').strip %>
+          <% formatted_date = created_date.sub(/\d+(st|nd|rd|th)/, '\1') %>
+          <p><%= formatted_date %></p>
+        </div>
+
+        <div class="col">
+          <% expense = category.expenses.first %>
+          <% if expense %>
+            <p><%= number_to_currency(expense.amount, unit: '$') %></p>
+          <% else %>
+            <p>No Expense</p>
+          <% end %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+  <%= link_to "New category", new_category_path, class: "btn btn-primary"  %>
```

Comment:
- [ ] From the design the new category button should be green with a padded white inner border and stick to the bottom of the page. Kindly implement that.

  Your Implementation | Design
  :---:|:---:
  ![image](https://github.com/Tracy-miranja/budget/assets/36542986/06ba6f09-3945-43cf-831b-ed9028d999c7) | ![image](https://github.com/Tracy-miranja/budget/assets/36542986/ff05278f-120b-484b-a568-0ac04c6d20ab)